src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Competitions/CompetitionForm.razor

```diff
@@ -26,24 +27,17 @@
     private MudPicker<TimeSpan?> _timeField = default!;
     private CompetitionFormModel _model { get; set; } = new();
 
-    public DateTimeOffset ToDateTimeOffSet(DateTime time)
-    {
-        DateTime timeWithSpecifiedKind = DateTime.SpecifyKind(time, DateTimeKind.Local);
-        DateTimeOffset offsetTime = timeWithSpecifiedKind;
-        return offsetTime;
-    }
     public DateTimeOffset CombineStartDayAndTime(DateTime? startDay, TimeSpan? startTime)
     {
         DateTime start = startDay.GetValueOrDefault(DateTime.MinValue);
-        DateTime startDayTime = start.Date.Add((TimeSpan)startTime);
-        DateTimeOffset startTimeOffset = ToDateTimeOffSet(startDayTime);
+        DateTime startDayTime = start.Date.Add(startTime.GetValueOrDefault(TimeSpan.Zero));
+        DateTimeOffset startTimeOffset = startDayTime.ToDateTimeOffset();
```

Comment:
I think we should either keep them nullable or set default values on the model object.

Reply:
Not sure I understand what you mean in this comment.